modules/introduction/partials/new-features-76.adoc

```diff
@@ -24,3 +24,8 @@ See xref:rest-api:rest-rotate-internal-credentials.adoc[Rotate Internal Credenti
 * By default, `cbbackupmgr` now backs up and restores user groups and users, including roles and permissions. User passwords are saved in the backup in a hashed format. When restoring a backup, `cbbackupmgr` defaults to not overwriting existing users in the database with identically named users in the backup. You can change these default `cbbackupmgr` behaviors using the new `--disable-users` and `--overwrite-users` command-line arguments. See  xref:backup-restore:cbbackupmgr-config.adoc[cbbackupmgr config] and xref:backup-restore:cbbackupmgr-restore.adoc[cbbackupmgr restore] for more about these new arguments.
 
 * The Role-Based Access Control (RBAC) REST API has a new `backup` endpoint that lets you backup and restore user and user groups. See xref:rest-api:rbac.adoc#backup-and-restore-users-and-groups[Backup and Restore Users and Groups]. 
+
+* You can choose to have a rebalance move an index's files between nodes instead of rebuilding them from scratch. 
```

Comment:
Suggested edit:

During rebalance, you can chose to move an index's files between nodes rather than rebuilding them from scratch.

Reply:
Updated with suggested edit.